sigopt/objects.py

```diff
@@ -284,3 +284,9 @@ class Token(ApiObject):
   client = Field(six.text_type)
   experiment = Field(six.text_type)
   user = Field(six.text_type)
+
+
+class AggregatedObservation(ApiObject):
+  assignments = Field(Assignments)
+  value = Field(float)
+  value_stddev = Field(float)
```

Comment:
i see... could a `failed` observation ever get returned by this endpoint? what if the user has only reported failed observations?